libs/components/src/next/form/radio/vl-radio-group.component.ts

```diff
@@ -0,0 +1,205 @@
+import { CSSResult, html, PropertyDeclarations, PropertyValues, TemplateResult } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import { radioStyle } from '@domg/govflanders-style/component';
+import { baseStyle, resetStyle } from '@domg/govflanders-style/common';
+import { vlElementsStyle } from '@domg-wc/elements';
+import radioUigStyle from './vl-radio.component.uig-css';
+import { FormControl, FormControlDefaults } from '../form-control/FormControl';
+import { VlRadioComponent } from '@domg-wc/components/next/form/radio/vl-radio.component';
+
+export const RadioGroupDefaults = {
+    ...FormControlDefaults,
+    block: false,
+    readonly: false,
+    value: '',
+};
+
+@customElement('vl-radio-group-next')
+export class VlRadioGroupComponent extends FormControl {
+    // Properties
+    private block = RadioGroupDefaults.block;
+    private readonly = RadioGroupDefaults.readonly;
+    private value: string | null = RadioGroupDefaults.value;
```

Comment:
| null is niet nodig als je een lege string gebruikt

Reply:
Gezien ik de `value` gelijkstel aan de laatste radio button die geklikt is (waar ik `.getAttribute('value')`) op aanroep, verwacht die dat `value` ook ` | null` kan zijn. Zie lijn 182 (functie `handleCheckedRadio`)